script/Arcueid/Day 7/07_00_ARC07_1C.txt

```diff
@@ -252,7 +253,7 @@ I'm at a loss for words.
 [sha:9e05c1e0648b8d50c41c58296b29837e1a61e188]{
 -- Page 108, Offset 16838.
 -- あいつは。ヴローヴは死んだ。この手で殺した。
-That guy is――― Vlov is dead. I killed him with my own hands.
+That guy is――― Vlov is dead. I killed him with my own two hands.
```

Comment:
this should be "em-dashes" with 2x cjk surrounding vlov imo. I see the Japanese only has a pause on one side but it reads kinda weird in english like this.